pyproject.toml

```diff
@@ -149,6 +150,15 @@ build-backend = "poetry.core.masonry.api"
 poethepoet = ">=0.26"
 poetry = ">=1.8"
 
+[tool.poe.tasks.build]
+help = "Build the package"
+shell = """
+make -C docs man
+rm -rf man
+mv docs/_build/man .
+poetry build
+"""
```

Comment:
I'm sure you tested this and made sure man build is now included in the package so also LGTM

Reply:
These instructions are also the same as the ones that were in `setup.py` previously :)